README.md

```diff
@@ -54,6 +56,17 @@ In addition to standards provided by [default](https://github.com/squizlabs/PHP_
 * WordPress-VIP
 * Zend
 
+### Custom Standards
+
+You can also include a custom phpcodesniffer standard configuration file. Add
+the file to the engine config standard section as a relative path:
+
+    engines:
+      phpcodesniffer:
+        enabled: true
+        config:
+          standard: "phpcs.ruleset.xml"
```

Comment:
Meh. I think it could merit its own section here since there's also a section specifically devoted to `standards` besides the default (PSR1 and PSR2) that we support but don't feel strongly.